Program.cs

```diff
@@ -0,0 +1,127 @@
+﻿using ShapesExercise;
+
+List<Shape> shapesList = new List<Shape>();
+Console.WriteLine("Enter 1. to create a new Shape");
+Console.WriteLine("Enter 2. to update a Shape");
+Console.WriteLine("Enter 3. to remove a Shape");
+Console.WriteLine("Enter 4. to print a Shape");
+int inputNumber = Convert.ToInt32(Console.ReadLine());
+while (inputNumber != -1)
+{
+    switch (inputNumber)
+    {
+        case 1:
+            Console.WriteLine("Enter 1. to create a Square");
+            Console.WriteLine("Enter 2. to create a Rectangle");
+            Console.WriteLine("Enter 3. to create a Rhombus");
+            Console.WriteLine("Enter 4. to create a Trapeze");
+            Console.WriteLine("Enter 5. to create a Triangle");
+            int shapeChoice = Convert.ToInt32(Console.ReadLine());
+            switch (shapeChoice)
+            {
+                case 1:
+                    Console.WriteLine("Enter edge length");
+                    int edgeSqaure = Convert.ToInt32(Console.ReadLine());
+                    Shape squareShape = new Square(edgeSqaure);
+                    shapesList.Add(squareShape);
```

Comment:
you can create a function in each shape class, that deals with creation of a new shape (with the logic you have here in each case).